cli/Cargo.toml

```diff
@@ -1,14 +1,15 @@
 [package]
 name = "phylum-cli"
 version = "3.5.0"
-authors = ["Eric Freitag <eric@phylum.io>"]
-edition = "2018"
+authors = ["Phylum, Inc. <engineering@phylum.io>"]
```

Comment:
👍 
Nothing against Eric, but this is the correct move...especially for a public repo.